{{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/variables.tf

```diff
@@ -43,6 +43,9 @@ variable "github_repo_url" {
 variable "github_server_url" {
   type        = string
   description = "URL of the hosted git server containing the current ML project, e.g. https://github.com/"
+  {% if cookiecutter.cicd_platform == "gitHub" -%}
+  default     = "https://github.com/"
+  {% endif -%}
```

Comment:
What happens if we don't specify this?

Reply:
I was manually prompted to add it in, which is weird because we already fill the other two variables as part of the command execution we recommend in the docs.